code/modules/busy_space/organizations.dm

```diff
@@ -0,0 +1,461 @@
+//Datums for different companies that can be used by busy_space
+/datum/lore/organization
+	var/name = ""				// Organization's name
+	var/short_name = ""			// Organization's shortname (Nanotrasen for "Nanotrasen Incorporated")
+	var/acronym = ""			// Organization's acronym, e.g. 'NT' for Nanotrasen'.
+	var/desc = ""				// One or two paragraph description of the organization, but only current stuff.  Currently unused.
+	var/history = ""			// Historical description of the organization's origins  Currently unused.
+	var/work = ""				// Short description of their work, eg "an arms manufacturer"
+	var/headquarters = ""		// Location of the organization's HQ.  Currently unused.
+	var/motto = ""				// A motto/jingle/whatever, if they have one.  Currently unused.
+
+	var/list/ship_prefixes = list()	//Some might have more than one! Like Nanotrasen. Value is the mission they perform, e.g. ("ABC" = "mission desc")
+	var/list/ship_names = list(		//Names of spaceships.  This is a mostly generic list that all the other organizations inherit from if they don't have anything better.
+		"Kestrel",
+		"Beacon",
+		"Signal",
+		"Freedom",
+		"Glory",
+		"Axiom",
+		"Eternal",
+		"Icarus",
+		"Harmony",
+		"Light",
+		"Discovery",
+		"Endeavour",
+		"Explorer",
+		"Swift",
+		"Dragonfly",
+		"Ascendant",
+		"Tenacious",
+		"Pioneer",
+		"Hawk",
+		"Haste",
+		"Radiant",
+		"Luminous"
+		)
+	var/list/destination_names = list()	//Names of static holdings that the organization's ships visit regularly.
+	var/autogenerate_destination_names = TRUE
+
+/datum/lore/organization/New()
+	..()
+	if(autogenerate_destination_names) // Lets pad out the destination names.
+		var/i = rand(6, 10)
+		var/list/star_names = list(
+			"Sol", "Alpha Centauri", "Sirius", "Vega", "Regulus", "Vir", "Algol", "Aldebaran",
+			"Delta Doradus", "Menkar", "Geminga", "Elnath", "Gienah", "Mu Leporis", "Nyx", "Tau Ceti",
+			"Wazn", "Alphard", "Phact", "Altair")
```

Comment:
Kaliban is the name of a character in a Fantastic Four comic and also is incredibly similar to Caliban, a character from Shakespeare's Tempest. Quite frankly, I assume it was probably not intended as a reference to either, but I think it is different enough where it won't get confused with Taliban